elixir/apps/web/lib/web/live/sites/new_token.ex

```diff
@@ -253,19 +253,18 @@ defmodule Web.Sites.NewToken do
         chmod +x /usr/local/bin/firezone-gateway; \\
       fi \\
     '
-    ExecStartPre=/usr/bin/mkdir -p /etc/firezone
-    ExecStartPre=/usr/bin/chown firezone:firezone /etc/firezone
-    ExecStartPre=/usr/bin/chmod 0755 /etc/firezone
-    ExecStartPre=/usr/bin/chmod +x /usr/local/bin/firezone-gateway
+    ExecStartPre=/bin/sh -c 'mkdir -p /etc/firezone'
+    ExecStartPre=/bin/sh -c 'chown firezone:firezone /etc/firezone'
+    ExecStartPre=/bin/sh -c 'chmod 0755 /etc/firezone'
+    ExecStartPre=/bin/sh -c 'chmod +x /usr/local/bin/firezone-gateway'
     AmbientCapabilities=CAP_NET_ADMIN
-    CapabilityBoundingSet=CAP_NET_ADMIN
     PrivateTmp=true
     ProtectSystem=full
     ReadWritePaths=/etc/firezone
     NoNewPrivileges=true
     TimeoutStartSec=15s
     TimeoutStopSec=15s
-    ExecStart=FIREZONE_NAME=$(hostname) /usr/local/bin/firezone-gateway
+    ExecStart=/bin/sh -c 'FIREZONE_NAME=$(hostname); sudo -u firezone -g firezone /usr/local/bin/firezone-gateway'
```

Comment:
I just realized, do we need to have some instructions to make sure they have created the `firezone` user in their OS?

Reply:
Its created in the unitfile above the download script

Reply:
Ah, ok